std/range/package.d

```diff
@@ -6730,6 +6734,381 @@ if (isForwardRange!Source && hasLength!Source)
     assert(equal(chunks, [[1], [2], [3], [], []]));
 }
 
+/**
+This range iterates over a fixed-sized sliding window (k-mers)
+of size $(D windowSize) of a $(D source) range.
+$(D Source) must be a forward range. $(D windowSize) must be
+greater than zero. For $(D windowSize = 2) it is similar to
+$D(zip(arr, arr.save.dropOne)).
+
+$(D[1,2,3].slides().array == [[1,2], [2,3]])
```

Comment:
This looks kind of out of place in the output and it's already in the example section, so maybe just remove it?